tsdb/compact_test.go

```diff
@@ -900,10 +900,9 @@ func BenchmarkCompactionFromHead(b *testing.B) {
 // This is needed for unit tests that rely on
 // checking state before and after a compaction.
 func TestDisableAutoCompactions(t *testing.T) {
-	db, closeFn := openTestDB(t, nil, nil)
+	db := openTestDB(t, nil, nil)
 	defer func() {
```

Comment:
`t.Cleanup(func() {` as well?

Reply:
Here there is no real added value, but I can do it also.

Reply:
I'd rather limit the scope at this for this PR, and look for other places where it would make sense.

Reply:
:+1: